contrib/kubespray/script/openpai_generator.py

```diff
@@ -233,6 +233,61 @@ def get_hived_config(layout, cluster_config):
     return { "skus": skus }
 
 
+def get_docker_cache_config_and_mirrors(layout, cluster_config):
```

Comment:
I think this function can be removed. During installation, using config.yml + services-configuration.yaml.template is enough.

Reply:
Removed